rclcpp/test/rclcpp/test_utilities.cpp

```diff
@@ -119,13 +119,6 @@ TEST(TestUtilities, test_context_basic_access) {
   EXPECT_EQ(std::string{""}, context1->shutdown_reason());
 }
 
-TEST(TestUtilities, test_context_basic_access_const_methods) {
-  auto context1 = std::make_shared<const rclcpp::contexts::DefaultContext>();
-
-  EXPECT_NE(nullptr, context1->get_init_options().get_rcl_init_options());
-  EXPECT_EQ(0u, context1->get_on_shutdown_callbacks().size());
-}
```

Comment:
I think we can keep this test, right?

Reply:
Yeah. I should keep get_rcl_init_options() test.